pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -43,8 +43,8 @@ social:
       %div.sw-tutorial-info
         %p.sw-tutorial-description= I18n.t(:starwars_blocks_description)
         %p.sw-tutorial-specs= I18n.t(:starwars_blocks_specs)
-      %a{:href => '#'}
-        %button.disabled.sw-disabled{disabled:"disabled"}= I18n.t(:starwars_blocks_start_button)
```

Comment:
The string for the javascript button is 'try it', which I wanted to use, although it could use better naming. I'll follow up with a string variable name change.

I restored the branch because Alice had me revert. Jael is working on the videos for this and we don't want to launch until the correct videos are in. Likely not until Friday.